static/js/services/unread-records.js

```diff
@@ -0,0 +1,86 @@
+var _ = require('underscore'),
+    readDocs = require('../modules/read-docs'),
+    TYPES = [ 'report', 'message' ];
+
+angular.module('inboxServices').factory('UnreadRecords', function(
+  $q,
+  Changes,
+  DB
+) {
+
+  'use strict';
+  'ngInject';
+
+  var getTotal = function() {
+    return DB().query('medic-client/data_records_by_type', { group: true });
+  };
+
+  var getRead = function() {
+    return DB({ meta: true }).query('medic-user/read', { group: true });
+  };
+
+  var getValue = function(response, type) {
```

Comment:
Is there a more descriptive function name we can use here?

Reply:
Fixed.